website/content/en/docs/building-operators/golang/advanced-topics.md

```diff
@@ -44,7 +44,7 @@ import (
 func init() {
 
     // Setup Scheme for all resources
-    utilruntime.Must(cachev1alpha1.AddToScheme(mgr.GetScheme()))
+    utilruntime.Must(cachev1alpha1.AddToScheme(scheme))
     // +kubebuilder:scaffold:scheme
```

Comment:
See that it has a text `Call the AddToScheme() function for your 3rd party resource and pass it the Manager's scheme via mgr.GetScheme()`. So, if we change the example we need to change the info. 

However, I am not sure about the change. Could you please clarify what is your motivation for that? Why `scheme` here would be better than `mgr.GetScheme()`? Did you make some test and it does not work as described?

Reply:
Both works fine, when I generate an operator, the init method already has this definition `utilruntime.Must(clientgoscheme.AddToScheme(scheme))` Just for the sake of consistency if I am adding a 3rd part. I can just pass the `scheme` same as the above case.

Reply:
Great. Tks for the answer. Should we not update the phase on top the snippets as well? See; https://github.com/operator-framework/operator-sdk/blob/2baf64888932a3384f2ba8e04bf797c778795540/website/content/en/docs/building-operators/golang/advanced-topics.md#register-with-the-managers-scheme

Reply:
sure, should I add like `pass it the Manager's scheme via scheme or mgr.GetScheme() in main.go`.

Reply:
I think it shows nice. For me, it is the only thing that it is missing to get my LGTM :-)

Reply:
done